src/ui/components/entityregions/EntityRegions.js

```diff
@@ -61,7 +66,7 @@ const EntityRegions = (props) => {
         )
         history.push({
             pathname: location.pathname,
-            search: queryString.stringify(newSearchParams)
+            search: queryString.stringify(newSearchParams, {arrayFormat: 'comma'})
```

Comment:
I added this so as for the visibility params to work after you maximise/contract a entityRegion. This did not break anything from my testing